### PR TITLE
updates-spring-2021

### PR DESCRIPTION
**Pull request by bogminic**

### CODE REVIEW

File: Week_02/Exercise/README.md [hunk 1]
```diff
@@ -1,378 +1,488 @@
...
+#### AC#2 – negative scenario
+
+**Given that:**
+
+* I am a Trader
+* AC#1 is in place
+
+**When** I try to filter based on multiple Currency Pairs at a time
+
+**Then** the system will not allow this action – once a new Ccy Pair value is selected, the previously selected one will get de-selected.
```
Comment (danamunteanu) [inline Week_02/Exercise/README.md:171]: de-selected or un-selected?

File: Week_02/Exercise/README.md [hunk 2]
```diff
@@ -1,378 +1,488 @@
...
+### USER STORY #4 – New User Register
 
-Business analysts enables an enterpries to articulate needs and the rationale for **change**, and to support design and describe solutions that can deliver value.
+**As an** unregistered user
 
-Business analysts are responsible for discovering, synthesizing and analyzing information from a variety of sources within an enterprise, including tools, processes, documentation and stakeholders.
+**I want to** be able to register in the FX Trading application
 
-## What does a BA do?
+**So that** I can perform my trending activity.
```
Comment (danamunteanu) [inline Week_02/Exercise/README.md:234]: 'my trading activity'

File: Week_03/Exercise/README.md [hunk 3]
```diff
@@ -1,91 +1,22 @@
...
-- action (sell/buy)
-- notional
-- tenor
-- transaction date/time
- 
-The user should be able to apply 3 filters to the dataset described above (currency pair, username, transaction date)
 ## Exercise
 
-Analyse the user requirements and based on them try and sketch a prototype for the three main pages. Get feedback from the user and improve your sketches until you reach the optimal solution for their needs.
+Analyse the [user requirements and stories](../../Week_02/Exercise/README.md) and based on them try and sketch a prototype for the three main pages. Get feedback from the user and improve your sketches until you reach the optimal solution for their needs.
```
Comment (danamunteanu) [inline Week_03/Exercise/README.md:14]: 'try to sketch'

**Review by danamunteanu — COMMENTED** [review]
Please see my comments and update the PR.

File: Week_02/Exercise/README.md [hunk 4]
```diff
@@ -1,378 +1,488 @@
...
+
+FX Trading application pages:
+
+### Register page
+
+A page that allows the creation of a new account.
+
+The page contains the following elements:
+
+page title: “Register a new account”
```
[the remark this answers is not in the document]
Comment (bogminic) [reply Week_02/Exercise/README.md:27]: Done

File: Week_02/Exercise/README.md [hunk 5]
```diff
@@ -1,378 +1,488 @@
...
+- “register” button
+- “Already have an account?” text that redirects the user to the login page
+
+### Login page
+
+A page that allows an user to login, giving them access to the FX Trading Dashboard.
+
+The page contains the following elements:
+
+Page title: “Login to your account”
```
[the remark this answers is not in the document]
Comment (bogminic) [reply Week_02/Exercise/README.md:42]: Done

File: Week_02/Exercise/README.md [hunk 6]
```diff
@@ -1,378 +1,488 @@
...
+
+- username field
+- password field
+- login button
```
Comment (bogminic) [inline Week_02/Exercise/README.md:46]: Done

File: Week_02/Exercise/README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (bogminic) [reply Week_02/Exercise/README.md:27]: Done

File: Week_02/Exercise/README.md (hunk 1 above)
Comment (bogminic) [inline Week_02/Exercise/README.md:171]: Done

File: Week_02/Exercise/README.md (hunk 2 above)
Comment (bogminic) [reply Week_02/Exercise/README.md:234]: Done

File: Week_03/Exercise/README.md (hunk 3 above)
Comment (bogminic) [reply Week_03/Exercise/README.md:14]: Done

**Review by danamunteanu — APPROVED** [review]

**Pull request merged**